markdown/assignments/pool_b/gameover/index.md

```diff
@@ -0,0 +1,24 @@
+---
+type: assignment
+title: "Game-Over"
+author: "Andre Matutat (FH Bielefeld)"
+points: 1
+weight: 3
+hidden: true
+---
+
+## Ziel
+
+In dieser Aufgabe implementieren Sie den Spieler Tot.
+
+## Voraussetzung
+
+Um die Aufgabe lösen zu können, brauchen Sie `["Monster"]({{< ref "/assignments/pool_b/monster" >}})`{=markdown} und ein beliebiges `["Kampfsystem"]({{< ref "/assignments/pool_b/nahkampf" >}})`{=markdown}.
```

Comment:
Wirklich "beliebig"? Du verlinkst auf den Nahkampf?!

Reply:
Alternativen:
1. Auf alle verlinken
2. Auf keins verlinken

Reply:
Ich finde das mit dem Verlinken erstmal gut. Ob das dauerhaft sinnvoll ist, müssen wir sehen (jede Änderung an den Strukturen bedeutet, in allen Markdown-Dateien die Verlinkung anzupassen). 

Insofern: Dann verlinke doch auf beide Kampfsysteme, oder?